### PR TITLE
Updated README with roadmap and gif example of usage

### PR DESCRIPTION
**Pull request by symphony-soufiane**
### Description
Added a simple example of usage with a gif attached to README.
Moved wdk presentation section from docs/readme to the root one.
Added a section with roadmap.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,16 +3,27 @@
...
 # Symphony Workflow Developer Kit (WDK)
 
-_Also known as Workflow API or Workflow bot_, check out the  [documentation](./docs).
+The Symphony Workflow Developer Kit (WDK) is a way to build bots on the Symphony platform with minimal coding efforts. A
+standard execution engine, provided as a bot is running workflows written in a declarative manner.
+
+Workflow executions are triggered by events (such as Datafeed events) and run activities. Activities are small building
+blocks often calling REST API endpoints of the Symphony platform or can even be custom-made to tailor your needs.
+
+- [Getting started](./getting-started.md): a beginner's guide to run your first workflow
```
Comment (symphony-thibault) [inline README.md:12]: I think it should be `./docs/getting-started.md`

File: README.md [hunk 2]
```diff
@@ -44,6 +55,13 @@ _NOTE:_ Commits and pull requests to FINOS repositories will only be accepted fr
 
 *Need an ICLA? Unsure if you are covered under an existing CCLA? Email [help@finos.org](mailto:help@finos.org)*
 
+## Roadmap
+- Support External message template
```
Comment (symphony-thibault) [inline README.md:59]: That would be nice to create GH issues (with a short description) for each roadmap item, and link them here.

File: README.md [hunk 3]
```diff
@@ -3,16 +3,27 @@
...
 :warning: _The WDK is very much in its inception and we are actually looking to get feedback on its usage. It also means
 all the APIs and the SWADL format are subject to change._
-
-## Getting started
-Check out the [Getting starting guide](./docs/getting-started.md) for an introduction.
+<hr />
 
 ## Usage example
-
+![](./gifs/wdk-demo.gif)
```
Comment (symphony-thibault) [inline README.md:26]: I'd prefer to move the `./gifs` folder inside `./docs` to let project root as light as possible

File: README.md [hunk 4]
```diff
@@ -3,16 +3,27 @@
 
 # Symphony Workflow Developer Kit (WDK)
 
-_Also known as Workflow API or Workflow bot_, check out the  [documentation](./docs).
+The Symphony Workflow Developer Kit (WDK) is a way to build bots on the Symphony platform with minimal coding efforts. A
+standard execution engine, provided as a bot is running workflows written in a declarative manner.
+
+Workflow executions are triggered by events (such as Datafeed events) and run activities. Activities are small building
```
Comment (symphony-thibault) [inline README.md:9]:
```suggestion
Workflow executions are triggered by events (such [Real-Time Events](https://docs.developers.symphony.com/building-bots-on-symphony/datafeed/real-time-events)) and run activities. Activities are small building
```

File: README.md (hunk 2 above)
Comment (symphony-soufiane) [reply README.md]: done

**Review by symphony-thibault — APPROVED** [review]
LGTM 👍

**Pull request merged**